### DISCUSSION TITLE
Reserved organisation scopes (e.g., @app)

### DISCUSSION THREAD

**Discussion by aral** [npm/feedback#905] [Registry]
Have some scopes been reserved by npm? (e.g., @app)

I see that there was a request for this at one point but that issue does not have a resolution: https://github.com/npm/npm/issues/20680

The reason I’m asking is because I’m going to be using `@app` as a scope prefix for app-specific local packages in [Kitten](https://codeberg.org/kitten/app) apps and I’d feel better knowing npm has reserved that handle (instead of some random person) and that it won’t be used in the future.

<strike>If you do a search for it currently (https://npmjs.com/org/app), it lists 0 packages and 0 organisations and the link forwards to (`/~app`). This is different behaviour to, for example, searching for https://npmjs.com/org/small-tech, which does not forward to a `~` url. https://npmjs.com/org/npm, however, also forwards to `/~npm`. So, is it correct for me to assume from this that this name has been reserved by npm?</strike>

(In which case, good.) :)

Thanks!

__Update:__ Ah, `~` is for accounts. e.g., `https://npmjs.com/org/aral` forwards to `/~aral`. So, no, I guess that “scope” hasn’t been reserved by npm. Please do correct me if I‘m wrong on that, though. Otherwise, please feel free to close this. I guess there won’t be a problem since local modules are installed via file URLs.

**Comment by ljharb** [comment]
As far as I know, there are no such reserved scopes. The best approach here imo is to choose a scope *you own publicly* - that way nobody can ever squat on it.
